delft/sequenceLabelling/wrapper.py

```diff
@@ -134,6 +134,14 @@ def __init__(self,
                                               early_stop, patience,
                                               max_checkpoints_to_keep, multiprocessing)
 
+        print("---")
+        print("max_epoch:", max_epoch)
+        print("early_stop:", early_stop)
+        print("batch_size:", batch_size)
+        print("max_sequence_length:", max_sequence_length)
+        print("model_name:", model_name)
+        print("---")
+
```

Comment:
We might want to add the `learning_rate` and `use-ELMo`? Because they are parameters that can impact quite a lot a training.